### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Apparently the README isn't clear enough about how this repo works (see #39). This PR tries to improve that.

I've already cleaned up the discussion sections, there's now only one left (discussions on announcements).

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Comment by felixfontein** [comment]
CC @webknjaz

**Review by Andersson007 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: false
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/config.yml:2]: @felixfontein do the announcement authors know to manually go to https://github.com/ansible-collections/news-for-maintainers/issues/new?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    By creating an issue you will spam a large amount of folks who will be annoyed if you post anything but a relevant announcement.
+    So **do not use this for anything but appropriate announcements!**
+- name: '**Discuss** an announcement'
+  url: https://github.com/ansible-collections/news-for-maintainers/discussions/new?category=general
+  about: If you want to **discuss an announcement**, use the Discussion feature of this repository. **Do not create an issue for this!**
+- name: 📝 Ansible **Code of Conduct**
+  url: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html?utm_medium=github&utm_source=issue_template_chooser
+  about: ❤ Be nice to other members of the community. ☮ Behave.
+- name: 💬 Talk to the community
+  url: https://docs.ansible.com/ansible/latest/community/communication.html?utm_medium=github&utm_source=issue_template_chooser#mailing-list-information
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/config.yml:17]: Looks like this anchor has changed over time:
```suggestion
  url: https://docs.ansible.com/ansible/latest/community/communication.html?utm_medium=github&utm_source=issue_template_chooser#asking-questions-over-email
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: false
+contact_links:
+- name: Create an **announcement**
+  url: https://github.com/ansible-collections/news-for-maintainers/issues/new
+  about: |
+    Use issues in this repositories only for announcements to collection maintainers.
+    By creating an issue you will spam a large amount of folks who will be annoyed if you post anything but a relevant announcement.
+    So **do not use this for anything but appropriate announcements!**
+- name: '**Discuss** an announcement'
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/config.yml:10]: How about aligning the style with other menu items?
```suggestion
- name: 💬 **Discuss** an announcement
```

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (webknjaz) [reply .github/ISSUE_TEMPLATE/config.yml:2]: Oh, I see there's a link below.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: false
+contact_links:
+- name: Create an **announcement**
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/config.yml:4]: How about making it tagged with smth like this?
```suggestion
- name: >-
    [ONLY FOR ADMINS]
    📢 Create an **announcement**
```
Also, maybe this item should be at the bottom of the list?

Comment (felixfontein) [reply .github/ISSUE_TEMPLATE/config.yml:4]: That would be wrong - anyone should be able to make announcements, not just admins or a selected group of folks.

Moving it to the bottom is probably a good idea though, to make it less likely to be abused.

**Pull request merged**